src/Symfony/Component/Validator/Tests/Fixtures/Entity.php

```diff
@@ -19,7 +19,7 @@
  * @Assert\GroupSequence({"Foo", "Entity"})
  * @Assert\Callback({"Symfony\Component\Validator\Tests\Fixtures\CallbackClass", "callback"})
  */
-class Entity extends EntityParent
+class Entity extends EntityParent implements EntityInterface, EntityParentInterface
```

Comment:
@HeahDude If I added `EntityParentInterface` to `EntityParent` would not have solved the issue because I had added the `EntityInterface` to it too. I think the new version is better.